lxcfs.c

```diff
@@ -514,7 +514,7 @@ static int cg_getattr(const char *path, struct stat *sb)
 		sb->st_nlink = 1;
 		sb->st_uid = k->uid;
 		sb->st_gid = k->gid;
-		sb->st_size = get_file_size(controller, path1, path2);
+		sb->st_size = 0;
```

Comment:
We can't return 0. FUSE filesystem is diferent from proc/sys filesystem which are based on [seq_file](https://www.kernel.org/doc/Documentation/filesystems/seq_file.txt).If return 0, kernel will return nothing.